src/parser.js

```diff
@@ -813,13 +813,6 @@ export default class Parser {
             this.position ++;
             let current = this.content();
             word += current;
-            if (current.lastIndexOf('\\') === current.length - 1) {
-                let next = this.nextToken;
-                if (next && next[TOKEN.TYPE] === tokens.space) {
-                    word += this.requiredSpace(this.content(next));
-                    this.position ++;
-                }
-            }
```

Comment:
No need this code anymore, we already handle this case (based on history https://github.com/postcss/postcss-selector-parser/commit/aded33a69fb19b9c6e28f3bdefbd5155754d3719)